README.md

```diff
@@ -75,6 +77,10 @@ For details, see: **[`Docker.md`](docs/Docker.md)**
 
 If you run into any errors [during the setup](docs/Getting-Started-frontend.md#how-do-i-get-set-up--just-run-the-site), try the suggestions in [`Troubleshooting.md`](docs/Troubleshooting.md).
 
+## Donating
+
+Donations to [support development on Group Income](https://groupincome.org/donate/) and the [other work of the okTurtles Foundation](https://okturtles.org/donate/) are greatly appreciated.
```

Comment:
Just one link to the okTurtles Foundation donation page here is fine. Text can say something like:

> Donations to the okTurtles Foundation go to supporting the development of Group Income and related projects